region/brinstar/green/Green Brinstar Beetom Room.json

```diff
@@ -287,17 +287,32 @@
     {
       "link": [1, 2],
       "name": "Jump Through the Room",
+      "entranceCondition": {
+        "comeInJumping": {
```

Comment:
This strat kinda depends on Beetom RNG tbh but I think just landing near the opposite door is enough to get through damageless.  
Jumping after the transition, with this tile count, lands on the opposite runway.
Jumping before the transition lands in the pit but the Beetoms avoid Samus anyway.